news.xml

```diff
@@ -306,14 +352,14 @@
       NixCon 2015
     </title>
     <description>
-      <a href="http://conf.nixos.org"><img class="inline" style="width: 10em;" src="https://d2z6c3c3r6k4bx.cloudfront.net/uploads/event/logo/1005856/banner.png" alt="NixCon logo"/></a>
+      <a href="http://conf.nixos.org"><img class="inline" style="width: 10em;"
+          src="https://d2z6c3c3r6k4bx.cloudfront.net/uploads/event/logo/1005856/banner.png" alt="NixCon logo" /></a>
       We're happy to announce that <strong>NixCon 2015</strong>, the
       first Nix Conference, will take place on <strong>November
-      14—15th 2015 in Berlin</strong>. For more information, see the
+        14—15th 2015 in Berlin</strong>. For more information, see the
       <a href="http://conf.nixos.org">NixCon website</a>. And please
-      consider <a
-      href="http://conf.nixos.org/submit-a-talk.html">submitting a
-      talk</a>!
+      consider <a href="http://conf.nixos.org/submit-a-talk.html">submitting a
```

Comment:
why was the identation changed here everywhere? it looks messed up

Reply:
i think my editor did this. need to somehow revert this.